cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -387,7 +387,7 @@ spec:
 
         # The logging level for the operator: INFO | DEBUG
         - name: ROOK_LOG_LEVEL
-          value: "INFO"
+          value: "DEBUG"
```

Comment:
Intentional?

Reply:
FWIW, I just execute a command after deploying the operator in my dev environment:
`kubectl --namespace rook-ceph set env deployment/rook-ceph-operator ROOK_LOG_LEVEL=DEBUG`